nipype/interfaces/afni/utils.py

```diff
@@ -1338,6 +1338,124 @@ def _list_outputs(self):
+             '               regression of the data from dataset1 vs. '
+             '               the dataset2'
+             '  * num      = number of the values in the region: '
+             '               with the use of -mask or -automask, '
+             '               the size of the region around any given '
+             '               voxel will vary; this option lets you '
+             '               map that size.'
+             '  * ALL      = all of the above, in that order'
+             'More than one option can be used.',
+        argstr='-stat %s')
+    mask_file = traits.File(
+        exists=True,
+        desc='mask image file name. Voxels NOT in the mask will not be used '
+             'in the neighborhood of any voxel. Also, a voxel NOT in the mask '
+             'will have its statistic(s) computed as zero (0).',
+        argstr='-mask %s')
+    automask = traits.Bool(
+        desc='Compute the mask as in program 3dAutomask.')
+    mask_file = traits.File(
+        exists=True,
+        desc='File name of an image to use as a weight.  Only applies to '
+             '\'pearson\' statistics.',
+        argstr='-weight %s',
+        xor=['automask'])
+    out_file = traits.File(
+        desc='Output dataset.',
+        argstr='-prefix %s',
+        name_source='in_files',
+        name_template='%s_bistat',
+        keep_extension=True,
+        position=0)
+
+
+class LocalBistat(AFNICommand):
+    """3dLocalBistat - computes statistics between 2 datasets, at each voxel,
+    based on a local neighborhood of that voxel.
+
+    For complete details, see the `3dLocalBistat Documentation.
+    <https://afni.nimh.nih.gov/pub../pub/dist/doc/program_help/3dLocalBistat.html>`_
+
+    Examples
+    ========
+
+    >>> from nipype.interfaces import afni
+    >>> bistat = afni.LocalBistat()
+    >>> bistat.inputs.in_files = ['functional.nii', 'structural.nii']
+    >>> bistat.inputs.neighborhood = ('SPHERE', 1.2)
+    >>> bistat.inputs.stat = 'pearson'
+    >>> bistat.inputs.outputtype = 'NIFTI'
+    >>> bistat.cmdline
+    "3dLocalBistat -prefix functional_bistat.nii -nbhd 'SPHERE(1.2)' -stat pearson functional.nii structural.nii"
+    >>> res = automask.run()  # doctest: +SKIP
+
+    """
+
+    _cmd = '3dLocalBistat'
+    input_spec = LocalBistatInputSpec
+    output_spec = AFNICommandOutputSpec
+
+    def _format_arg(self, name, spec, value):
+        if name == 'neighborhood':
+            region_name, region_size = value
+            if region_name == 'RECT':
+                return spec.argstr % (
+                    "'{0}({1})'".format(region_name, ','.join(region_size)))
+            else:
+                return spec.argstr % (
+                    "'{0}({1})'".format(region_name, region_size))
+        if name == 'stat':
+            if isinstance(value, (str, bytes)):
+                return spec.argstr % value
+            else:
+                return ' '.join([spec.argstr % v for v in value])
```

Comment:
>I'm pretty sure you can drop this if name == 'stat' block and get the same behavior.

this  doesn't seem to work: it doesn't repeat the option '-stat' before each value
```Python
bistat.inputs.stat = ['pearson', 'ALL']
```
gives `-stat pearson ALL` instead of `-stat pearson -stat ALL`

Reply:
@salma1601 - if you change the `argstr` above to `'-stat %s...'` it will do what you are looking for.